### PR TITLE
Release 1.4.1

### PR DESCRIPTION
**Pull request by dkliban**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 setup(
     name='pulp-container',
-    version='1.4.1.dev',
+    version='1.4.2.dev',
```
Comment (evgeni) [inline setup.py:13]: shouldn't this (and all other places) say "1.4.1"? :)

Comment (evgeni) [reply setup.py:13]: so, essentially, drop 3631159ed2ebfb1b3b2c0582ddd629f3d4c72440 from this PR :)

Comment (dkliban) [reply setup.py:13]: There are multiple commits here. There is an intermediate commit with 1.4.1 and the changelog.

Comment (evgeni) [reply setup.py:13]: from irc:

```
< Zhenech> so, you'll be tagging the intermediate?
< dkliban> that's right
< Zhenech> ewwww @ workflow
```

so shutting up :)

**Pull request merged**